web/themes/custom/cloudy/scss/_overrides.scss

```diff
@@ -3,14 +3,34 @@ Theme Name: Bootstrap 4 Sass
 Description: Bootstrap 4 with Sass
 */
 
+/* TYPOGRAPHY */
+
+/* Google Fonts */
+@import url("//fonts.googleapis.com/css?family=Open+Sans:300,400,300italic,400italic");
+
 /* GENERAL STYLES
 -------------------------------------------------*/
-
 body {
     background: $white;
-    font-weight: 300;
+    font-weight: $font-weight-light;
```

Comment:
I don't see this variable declared anywhere in our theme or Bootstrap, but I do see it showing up in style.css as a class. Am I missing something?

Reply:
$font-weight-light is defined in the bootstrap _variables.scss